apps/docs/app/ui/layout/header/Header.tsx

```diff
@@ -1,20 +1,100 @@
 "use client";
 
+import { useContext, useEffect, useState, useRef, type ReactNode } from "react";
 import Link from "next/link";
+import { SlotProvider } from "@hopper-ui/components";
 
 import ThemeSwitch from "@/components/themeSwitch/ThemeSwitch";
 import IconButton from "@/components/iconButton/IconButton";
+import { Icon, ProductMenuIcon } from "@/components/icon";
+import { ToggleButton, ToggleButtonContext } from "@/components/toggleButton/ToggleButton.tsx";
 import MobileMenu from "@/app/ui/layout/mobileMenu/MobileMenu";
 import MobileMenuTrigger from "@/app/ui/layout/mobileMenu/MobileMenuTrigger";
 import Nav from "@/app/ui/layout/nav/Nav";
 import Wrapper from "@/app/ui/layout/wrapper/Wrapper";
+import { Popover, PopoverTrigger, PopoverContext } from "@/components/popover/Popover";
+import { useIsMobile } from "@/hooks/useIsMobile";
 import { navigation } from "@/configs/navigation";
-import { useContext, useEffect, useState } from "react";
+import { type ColorScheme, ThemeContext } from "@/context/theme/ThemeProvider.tsx";
 
 import HopperLogo from "./assets/hopper-logo.svg";
+
 import "./header.css";
-import { type ColorScheme, ThemeContext } from "@/context/theme/ThemeProvider.tsx";
-import { useIsMobile } from "@/hooks/useIsMobile";
+
+
+const designSystemLinks = [
+    {
+        title: "Orbit",
+        url: "https://orbit.sharegate.design/?path=/docs/installation--page"
+    },
+    {
+        title: "Orbiter",
+        url: "https://wl-orbiter-website.netlify.app/?path=/docs/installation--page"
+    },
+    {
+        title: "Igloo",
+        url: "https://igloo.officevibe.design"
+    }
+];
+
+const ToggleTrigger = ({ children }: { children: ReactNode }) => {
+    const triggerRef = useRef(null);
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+    return (
+        <SlotProvider values={[
+            [PopoverContext, {
+                isOpen: isPopoverOpen,
+                onOpenChange: setIsPopoverOpen,
+                placement: "bottom start",
+                offset: 20
+            }],
+            [ToggleButtonContext, {
+                onPress: () => setIsPopoverOpen(!isPopoverOpen),
+                ref: triggerRef,
+                isSelected: isPopoverOpen
+            }]
+        ]}
+        >
+            {children}
+        </SlotProvider>
+    );
+};
+
+const LeftNav = () => {
```

Comment:
This could be renamed. As naming based position in the DOM is bound to not always be true.